subscription/downgrade.md

```diff
@@ -7,36 +7,30 @@ redirect_from:
 - /docker-hub/billing/downgrade/
 ---
 
-You can downgrade your Docker subscription anytime. When you downgrade your subscription, changes are applied immediately, and you will no longer have access to the prior services.
+You can downgrade your Docker subscription at anytime before the renewal date. The unused portion of the subscription isn't refundable.
 
-For example, if you are currently on a **Docker Business** subscription and want to downgrade to a **Docker Team**, your plan will change immediately. You will no longer have access to your previous service, such as Single Sign-on.
+When you downgrade your subscription, changes are applied immediately and you no longer have access to prior services.
 
-The following sections contain instructions on how to downgrade your existing Docker subscription.
+Before you downgrade to a free plan, you may need to reduce the number of team members and convert any private repositories to public repositories. For information, see [Docker Pricing](https://www.docker.com/pricing){: target="_blank" rel="noopener" class="_"}.
```

Comment:
**suggestion** let's keep using `subscription` vs `plan` to be consistent with the language in the subscription details section.

```suggestion
Before you downgrade to a free subscription, you may need to reduce the number of team members and convert any private repositories to public repositories. For information, see [Docker Pricing](https://www.docker.com/pricing){: target="_blank" rel="noopener" class="_"}.
```